roles/users/tasks/add_sudoers.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: check to make sure sudo is installed
+  apt:
+    pkg=sudo state=present update_cache=yes
+  tags: sudo
+ 
+- name: configure sudoers
+  lineinfile: dest=/etc/sudoers.d/{{item}} owner=root group=root mode=0440
+              line="{{item}} ALL=(ALL) NOPASSWD:/usr/sbin/service"
```

Comment:
I would not do a line for all services, this will allow the users to do too many modifications, could you just give access to `/etc/init.d/apache` and `/etc/init.d/varnish`